usaspending_api/api_contracts/contracts/references/Glossary.md

```diff
@@ -0,0 +1,83 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Glossary
+
+These endpoints are used to power USAspending.gov's glossary components. 
+
+
+
+## List Glossary [/api/v2/references/glossary/{?limit,page}]
+
+This endpoint returns a list of glossary data.
+
++ Parameters
+    + `page`: `1` (optional, integer)
+        Page number to return
+    + `limit`: `10` (optional, integer)
+        Maximum number to return
+
+### List Glossary [GET]
+
++ Response 200 (application/json)
+    + Attributes (GlossaryListingObject)
+
+## Glossary Autocomplete [/api/v2/autocomplete/glossary/]
+
+This endpoint returns glossary autocomplete data for submitted text snippet.
+
+
++ Parameters
+    + search_text: `ab` (required, string)
+        The text snippet that you are trying to autocomplete using a glossary term.
+    + limit: `10` (optional, integer)
+        Maximum number to return
+
+### List Autocomplete Glossary [POST]
+
++ Request (application/json)
+
+    + Attributes
+
+        + search_text: `ab` (required, string)
+        The text snippet that you are trying to autocomplete using a glossary term.
+        + limit: `10` (optional, number)
+        Maximum number to return
+        
++ Response 200 (application/json)
+    + Attributes (GlossaryAutocompleteObject)
```

Comment:
The v1 response had some unnecessary nesting and unclear naming conventions that I think we have the chance to improve here. I would reorganize this response to have the `search_text` and count of matched terms at the top level, along with an array of `GlossaryListing` objects. Something like:
```
{
	"search_text": "ab",
	"count": 4,
	"matched_terms": [{
		"term": "Availability Type Code",
		"slug": "availability-type-code",
		"data_act_term": "Availability Type Code",
		"plain": "Within a Treasury Account Symbol (TAS), this one-letter code Identifies the availability (or time period) for obligations to be made on the appropriation account. A TAS will have an “X” if there is an unlimited or indefinite period to incur new obligations.",
		"official": "In appropriations accounts, the availability type code identifies an unlimited period to incur new obligations; this is denoted by the letter X.",
		"resources": null
	}, ...],
	"results": [
		"Availability Type Code",
		"Labor Standards", ...
	]
}
```